src/EditorFeatures/Core/Implementation/TextStructureNavigation/AbstractTextStructureNavigatorProvider.TextStructureNavigator.cs

```diff
@@ -69,7 +69,7 @@ private TextExtent GetExtentOfWordWorker(SnapshotPoint position, CancellationTok
                 // If at the end of the file, go back one character so stuff works
                 if (position == textLength && position > 0)
                 {
-                    position = position - 1;
+                    position -= 1;
```

Comment:
>position -= 1; [](start = 20, length = 14)

`position--;` ?

Reply:
Yes.  it would def be good for the feature to detect this pattern and suggest -- and ++ accordingly.  But i'll treat that as more of a feature suggestion :)